src/cmd/top/joblist_pane.c

```diff
@@ -104,6 +150,29 @@ static void joblist_continuation (flux_future_t *f, void *arg)
     flux_future_destroy (f);
 }
 
+void joblist_pane_enter (struct joblist_pane *joblist)
+{
+    struct top *top;
+    char *uri = NULL;
+    json_t *job = get_current_job (joblist);
+    if (!job)
+        return;
+    if (json_unpack (job,
+                     "{s:{s:{s:s}}}",
+                     "annotations",
+                       "user",
+                         "uri", &uri) < 0)
+        return;
+    if (uri == NULL)
+        return;
+    if (!(top = top_create (uri)))
+        fatal (errno, "failed to create sub-top for job");
```

Comment:
Does this mean browsing the system instance and selecting another guest's batch job is fatal to top?
It might be nice if the interface didn't allow those jobs to be selected, or if the error was non fatal.  Not sure what that should like like in the UI...

Reply:
Yeah, that's a good point. For now it behaves like you ran `flux top JOBID`. 

I can try to pop up a dialog box, I'm sure curses must have support for that. Or for now we could have "nothing silently happen" like it does for non-instance jobs :laughing: (I thought maybe in the future hitting enter on a non-instance job could attach to it in a new window or something, and there could be other commands, like get more detailed information, jobspec browser, cancel, etc.)